app/lib/actions/recurring_logic/create.rb

```diff
@@ -0,0 +1,17 @@
+module Actions
+  module RecurringLogic
+    class Create < EntryAction
+
+      middleware.use Actions::Middleware::InheritTaskGroups
+
+      def plan(recurring_logic)
+        recurring_logic.task_group = ::ForemanTasks::TaskGroups::RecurringLogicTaskGroup.new
```

Comment:
Why separate action for this? Shouldn't this be part of the model (before create?)